website/scripts/app/main_menu_bottom.js

```diff
@@ -134,30 +143,102 @@ $(document).ready(function(){
 
      $('#speedAdvisoryInfoInfoCloseBtn').click(function(){
           $("#speed-advisory-info-panel").css('display','none');
-          $("#speed-advisory-info-btn").css('background-color','black');
-          $("#speed-advisory-info-btn").css('color','white');
+          $("#geofence-info-btn").css('background-color','black');
+          $("#geofence-info-btn").css('color','white');
      });
 
      $('#SystemStatusInfoInfoCloseBtn').click(function(){
           $("#status-panel").css('display','none');
           $("#status-btn").toggleClass('active');
      });
 
-     $('#SystemLogsInfoInfoCloseBtn').click(function(){
+     $('#SystemLogsInfoInfoCloseBtn').click(function()
+     {
           $("#logs-panel").css('display','none');
           $("#logs-btn").toggleClass('active');
      });
 
-     $('#ActivePluginInfoInfoCloseBtn').click(function(){
+     $('#ActivePluginInfoInfoCloseBtn').click(function()
+     {
           $("#activePlugins-panel").css('display','none');
           $("#activePlugins-btn").toggleClass('active');
      });
 
-     $('#MapsInfoInfoCloseBtn').click(function(){
+     $('#MapsInfoInfoCloseBtn').click(function()
+     {
           $("#maps-panel").css('display','none');
           $("#maps-btn").toggleClass('active');
      });
      
+     //Check dropdown every 10 seconds
```

Comment:
I don't think you have to check every 10 seconds. The "Platoon Info" and "Geofence Advisory" menu items should be based on the selection, and it would be final after they hit the C-ADS button and the guidance has been set to ACTIVE/ENGAGED. This selection will remain until they restart guidance.

Reply:
It actually hide/clear the dropdown every 10 seconds, so that we get the latest dropdown items. Geofence Advisory is based on whether the vehicle reach the geofence zone.

Reply:
@dan-du-car i keep forgetting the geofence advisory is not a plugin. And the platoon info should be fixed on your Platoon info task.